.travis.yml

```diff
@@ -17,7 +17,7 @@ env:
   global:
     - CRDS_SERVER_URL='https://jwst-crds.stsci.edu'
     - CRDS_PATH='/tmp/crds_cache'
-    - CRDS_CONTEXT="jwst_0508.pmap"
+    - CRDS_CONTEXT="jwst_0513.pmap"
```

Comment:
This line can be removed entirely.  Normally we do not set the CRDS_CONTEXT for our CI unit tests.  We just use the default.

Reply:
Done